### PR TITLE
Added vehicle keys to the motorbike

### PR DESCRIPTION
**Pull request by lbnesquik**
## About The Pull Request
Fixed a typo in the cooldowns
Added key sprites
Expanded a check so you can remove vehicle keys if no one is riding them, including yourself
Made the motorbike use keys
## Why It's Good For The Game
Your precious baby bike will not so easily stolen now. 
## Changelog
:cl:
add: Added vehicle keys to the motorbike
qol: Expanded a check so you can remove vehicle keys if no one is riding them, including yourself
/:cl:

### CODE REVIEW

**Comment by Lumipharon** [comment]
But, why?

**Comment by lbnesquik** [comment]
People in the discord kept complaining about their bike getting stolen.

File: code/modules/vehicles/motorbike.dm [hunk 1]
```diff
@@ -53,6 +54,9 @@
 	return welder_repair_act(user, I, 10, 2 SECONDS, fuel_req = 1)
 
 /obj/vehicle/ridden/motorbike/relaymove(mob/living/user, direction)
+	if(!inserted_key)
+		to_chat(user, span_warning("You need the key to use the bike!"))
+		return
```
Comment (TiviPlus) [inline code/modules/vehicles/motorbike.dm:59]: Shouldn't this be on relay drive or something to fix this?

File: code/modules/vehicles/motorbike.dm [hunk 2]
```diff
@@ -124,6 +128,9 @@
 		max_buckled_mobs = 2
 		max_occupants = 2
 		return TRUE
+
+	if(istype(I, /obj/item/key/motorbike))
+		return
```
Comment (TiviPlus) [inline code/modules/vehicles/motorbike.dm:133]: This should check key type

File: code/modules/vehicles/ridden.dm [hunk 3]
```diff
@@ -42,7 +42,7 @@
 /obj/vehicle/ridden/AltClick(mob/user)
 	if(!inserted_key || (!user.Adjacent(src) && user.dextrous))
 		return ..()
-	if(!is_occupant(user))
+	if(!is_occupant(user) && occupants)
```
Comment (TiviPlus) [inline code/modules/vehicles/ridden.dm:45]: ?

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/modules/vehicles/ridden.dm (hunk 3 above)
Comment (lbnesquik) [reply code/modules/vehicles/ridden.dm:45]: I added it so you can remove the key without standing on the bike, if no one is riding it.

File: code/modules/vehicles/motorbike.dm (hunk 1 above)
Comment (lbnesquik) [reply code/modules/vehicles/motorbike.dm:60]: Well, given there's the fuel check just below, I figured it was fine

File: code/modules/vehicles/motorbike.dm (hunk 2 above)
Comment (lbnesquik) [reply code/modules/vehicles/motorbike.dm:134]: Done

Comment (lbnesquik) [reply code/modules/vehicles/motorbike.dm]: Done

**Pull request closed without merging**